### ISSUE TITLE
Install buttons on discovery pane don't show up in high contrast mode

### ISSUE THREAD

**Issue by aswan** [mozilla/addons#11599]
### Describe the problem and steps to reproduce it:
Put the browser into high contrast mode (Preferences → General → Colors → Override the colors specified by the page with your selections above "Always") and visit the discovery pane

### What happened?
The interior part of the install buttons on the discovery pane are not visible:
https://bug1445552.bmoattachments.org/attachment.cgi?id=8958689

### What did you expect to happen?
For them to be visible I suppose

### Anything else we should know?
this was originally filed at https://bugzilla.mozilla.org/show_bug.cgi?id=1445552
